lib/fix-me.js

```diff
@@ -18,10 +18,10 @@ FixMe.prototype.runEngine = function(){
       self = this;
 
   if (fs.existsSync('/config.json')) {
-    var overrides = JSON.parse(fs.readFileSync('/config.json'));
-
-    for (var prop in overrides) {
-        config[prop] = overrides[prop];
+    var user_config = JSON.parse(fs.readFileSync('/config.json'));
```

Comment:
nit: we generally use snakeCasing for var names in JS (except in cases where it's a key that maps to JSON or something).